frontend/lib/resolution/resolution-types.cpp

```diff
@@ -447,21 +457,23 @@ CallInfo CallInfo::create(Context* context,
       // a value (ambiguity or other "benign" UNKNOWN would not produce errors).
       // Later, this can lead to skipping resolving the call altogether.
 
-      name = USTR("this");
-      // add the 'this' argument as well
-      isMethodCall = true;
-      actuals.push_back(CallInfoActual(*calledExprType, USTR("this")));
-      if (actualAsts != nullptr) {
-        actualAsts->push_back(calledExpr);
-      }
+      makeMethodCall();
```

Comment:
I think a name mentioning the "this" aspect would be better. E.g., "makeCallToThis"?